agent/engine/task_manager_test.go

```diff
@@ -139,6 +139,18 @@ func TestHandleEventError(t *testing.T) {
 			ExpectedContainerDesiredStatusStopped: true,
 			ExpectedOK:                            false,
 		},
+		{
+			Name:                        "Start EOF",
```

Comment:
nit: more descriptive naming for these test cases please

Reply:
sure, updated